reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -209,21 +209,9 @@ class StaticHtmlReporter : Reporter {
         }
     }
 
-    private val PathExclude.description: String get() =
-        buildString {
-            append(reason)
-            if (comment.isNotBlank()) {
-                append(" - ").append(comment)
-            }
-        }
+    private val PathExclude.description: String get() = joinNonBlank(reason.toString(), comment)
 
-    private val ScopeExclude.description: String get() =
-        buildString {
-            append(reason)
-            if (comment.isNotBlank()) {
-                append(" - ").append(comment)
-            }
-        }
+    private val ScopeExclude.description: String get() = joinNonBlank(reason.toString(), comment)
```

Comment:
Just wondering: What's the motivation to use two times the same private extension function (also in ExcelReporter) instead of simply making this a property or function in the Path- / ScopeExclude classes themselves? If the only motivation is to exclude them from being serialized, I guess I'd prefer a `@JsonIgnore`.

Reply:
I basically agree with that motivation (although I find that it increases complexity as code becomes mode scattered, but it's worth that cost). However, if we also agree that this is the way reporters should display the description, this should be common to all reporters (or at least *available* to all reporters) and thus these extension functions should be moved to a common place, probably to the file of the `Reporter` base class, with `internal` visibility.

Reply:
My preference is different, but only slightly - so let's do as propose. I'll make a PR.

Reply:
@sschuberth here it is: #2586.